test/solver_tests/snj_solver_test.py

```diff
@@ -1,14 +1,14 @@
 """
 Test SpectralNeighborJoiningSolver in Cassiopeia.solver.
 """
-import unittest
-from typing import List
-from unittest import mock
 import itertools
 import networkx as nx
 from networkx.classes.digraph import DiGraph
 import numpy as np
 import pandas as pd
+from typing import List
+import unittest
+from unittest import mock
```

Comment:
```
from typing import List

import itertools
import networkx as nx
import numpy as np
import pandas as pd
import unittest

from unittest import mock
from networkx.classes.digraph import DiGraph
```

Reply:
Hey Kevin and Ivan, mind doing the imports like this? Essentially, we place the "from" statements afterwards, except for the imports that are needed for "basic functions" like typing